lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -20,11 +20,40 @@ import 'package:over_react_codemod/src/util.dart';
 typedef YieldPatch = void Function(
     int startingOffset, int endingOffset, String replacement);
 
-@visibleForTesting
-bool isPublicForTest = false;
+SemverHelper semverHelper;
 
-// Stub while <https://jira.atl.workiva.net/browse/CPLAT-9308> is in progress
-bool isPublic(ClassDeclaration node) => isPublicForTest;
+/// Returns whether or not [node] is publicly exported.
+bool isPublic(ClassDeclaration node) {
+  assert(semverHelper != null);
+  return semverHelper.getPublicExportLocations(node) != null;
+}
+
+class SemverHelper {
+  Map _exportList;
+
+  SemverHelper(Map jsonReport) {
+    _exportList = jsonReport['exports'];
```

Comment:
#nit According to the Dart style guide, [initializing formals are preferred](https://dart.dev/guides/language/effective-dart/usage#do-use-initializing-formals-when-possible) over initializing fields in the constructor body, and [final fields are also preferred when possible](https://dart.dev/guides/language/effective-dart/design#prefer-making-fields-and-top-level-variables-final).

So, this would become:
```dart
final Map _exportList;

SemverHelper(Map jsonReport) : _exportList = jsonReport['exports'];
```